src/main/java/io/ebeaninternal/server/deploy/parse/AnnotationFields.java

```diff
@@ -282,12 +297,8 @@ private void readField(DeployBeanProperty prop) {
       generatedPropFactory.setUpdateTimestamp(prop);
     }
 
-    if (get(prop, WhoCreated.class) != null) {
-      generatedPropFactory.setWhoCreated(prop);
-    }
-    if (get(prop, WhoModified.class) != null) {
-      generatedPropFactory.setWhoModified(prop);
-    }
+    initWhoCreated(prop);
+    initWhoModified(prop);
 
```

Comment:
I'd say just leave it.  I'll merge it in shortly and square it away - all good.  
Cheers, Rob.